sdk/iot/Azure.Iot.Hub.Service/src/API Design.md

```diff
@@ -107,7 +107,65 @@ APIs for managing configurations for devices and modules.
 <details>
 
 ```csharp
+/// <summary>
+/// Gets a configuration on the IoT Hub for automatic device/module management
+/// </summary>
+/// <param name="configurationId">The unique identifier of the configuration.</param>
+/// <param name="cancellationToken">The cancellation token.</param>
+/// <returns>TwinConfiguration for a automatic device/module management</returns>
```

Comment:
an automatic

Reply:
Although even after that, I don't follow what an "automatic device/module management" is. Shouldn't it be something like an "automatic device/module management rule" or something?

Reply:
"TwinConfiguraton to be used for automatic device/module management"?

Reply:
or "Configuration to be used for automatic device/module management"

Reply:
I am not sure if adding "to be used" instead of "for" makes it any easier. However I don't mind making a change we can all agree to one suggestion that would be easier from customer perspective.

Reply:
On a second thought - how about "The retrieved twin configuration for an automatic device/module management" since that's the pattern we follow for other clients

Reply:
"an automatic device/module management" isn't proper english. "The retrieved automatic device/module management twin configuration" would make more sense

Reply:
I still find it hard to read as it makes too long. I have made the changes though.